qa/tasks/cephfs/test_volumes.py

```diff
@@ -921,6 +924,36 @@ def test_subvolume_create_with_desired_data_pool_layout_in_group(self):
         # verify trash dir is clean
         self._wait_for_trash_empty()
 
+    def test_subvolume_create_with_desired_mode(self):
+        subvol1 = self._generate_random_subvolume_name()
+
+        # default mode
+        expected_mode1 = "755"
```

Comment:
Why not just call the variable `default_mode`?

Reply:
Done